tests/MatrixDotNet.UnitTesting/MatrixTests/BasicOperations.cs

```diff
@@ -87,6 +88,64 @@ public void MatrixMultiplySimdDoubleTest_MultiplyTwoMatrix_AssertMustBeEqual()
             // Assert
             Assert.Equal(expected, actual);
         }
+
+        [Theory]
+        [InlineData(8)]
+        [InlineData(16)]
+        [InlineData(17)]
+        [InlineData(21)]
+        [InlineData(32)]
+        public void MatrixMultiply_CanMultiply(int size)
+        {
+            // Arrange
+            var matrixA = new Matrix<double>(size, size, 1);
+            var matrixB = new double[size, size];
+            for (int i = 0; i < matrixB.GetLength(0); i++)
+            {
+                for (int j = 0; j < matrixB.GetLength(1); j++)
+                {
+                    matrixB[i, j] = 1;
+                }
+            }
+            
+            var expected = MulMatrixTest(matrixB, matrixB);
```

Comment:
Multiplication result for square matrixes that consist only of `1` will be just matrix `size`x`size` that consist of `size`